internal/common/types.go

```diff
@@ -115,14 +115,16 @@ type Config struct {
 	Registry RegistryService
 	// Clients is a map of services used by a DS.
 	Clients map[string]ClientInfo
-	// Device contains device-specific coniguration settings.
+	// Device contains device-specific configuration settings.
 	Device DeviceInfo
 	// Logging contains logging-specific configuration settings.
 	Logging LoggingInfo
 	// Watchers is a map provisionwatchers to be created on startup.
 	Watchers map[string]WatcherInfo
 	// DeviceList is the list of pre-define Devices
 	DeviceList []DeviceConfig `consul:"-"`
+	// Driver is a map for protocol driver's customizable configuration
```

Comment:
rephrase
Driver is a string map contains customized configuration for the protocol driver using Device SDK

Reply:
Rephrased